samples/sample_ecu_manifest.json

```diff
@@ -1 +1,25 @@
-{"signatures": [{"method": "ed25519", "sig": "df043006d4322a386cf85a6761a96bb8c92b2a41f4a4201badb8aae6f6dc17ef930addfa96a3d17f20533a01c158a7a33e406dd8291382a1bbab772bd2fa9804", "keyid": "49309f114b857e4b29bfbff1c1c75df59f154fbc45539b2eb30c8a867843b2cb"}], "signed": {"timeserver_time": "2016-10-14T16:06:03Z", "installed_image": {"filepath": "/file2.txt", "fileinfo": {"hashes": {"sha256": "3910b632b105b1e03baa9780fc719db106f2040ebfe473c66710c7addbb2605a", "sha512": "e2ebe151d7f357fcc6b0789d9e029bbf13310e98bc4d15585c1e90ea37c2c7181306f834342080ef007d71439bdd03fb728186e6d1e9eb51fdddf16f76301cef"}, "length": 21}}, "previous_timeserver_time": "2016-10-14T16:06:03Z", "ecu_serial": "ecu11111", "attacks_detected": ""}}
\ No newline at end of file
+{
+ "signatures": [
+  {
+   "keyid": "49309f114b857e4b29bfbff1c1c75df59f154fbc45539b2eb30c8a867843b2cb",
+   "method": "ed25519",
+   "sig": "fd04c1edb0ddf1089f0d3fc1cd460af584e548b230d9c290deabfaf29ce5636b6b897eaa97feb64147ac2214c176bbb1d0fa8bb9c623011a0e48d258eb3f9108"
+  }
+ ],
+ "signed": {
+  "attacks_detected": "",
+  "ecu_serial": "TCUdemocar",
+  "installed_image": {
+   "fileinfo": {
+    "hashes": {
+     "sha256": "6b9f987226610bfed08b824c93bf8b2f59521fce9a2adef80c495f363c1c9c44",
+     "sha512": "706c283972c5ae69864b199e1cdd9b4b8babc14f5a454d0fd4d3b35396a04ca0b40af731671b74020a738b5108a78deb032332c36d6ae9f31fae2f8a70f7e1ce"
+    },
+    "length": 37
+   },
+   "filepath": "/secondary_firmware.txt"
+  },
+  "previous_timeserver_time": "2017-05-18T16:37:46Z",
+  "timeserver_time": "2017-05-18T16:37:48Z"
```

Comment:
I'm not sure how these timestamps are validated, but should they be set to expire far into the the future to avoid issues with invalid ECU manifests or timeserver attestations?